cmd/admin-bucket-remote-bandwidth.go

```diff
@@ -140,10 +140,10 @@ func checkAdminBwInfoSyntax(ctx *cli.Context) {
 		u != "Bi" &&
 		u != "B" &&
 		u != "" {
-		showCommandHelpAndExit(ctx, "bandwidth", globalErrorExitStatus)
```

Comment:
you don't need to do that I have a more comprehensive change #4338

Reply:
ok,copy.